packages/react-composites/review/react-composites.api.md

```diff
@@ -753,6 +770,62 @@ export type TopicChangedListener = (event: {
     topic: string;
 }) => void;
 
+// @beta (undocumented)
+export const UPLOAD_CANCELLED_EVENT = "uploadCancelled";
+
+// @beta (undocumented)
+export const UPLOAD_COMPLETED_EVENT = "uploadCompleted";
+
+// @beta (undocumented)
+export const UPLOAD_FAILED_EVENT = "uploadFailed";
+
+// @beta (undocumented)
+export const UPLOAD_PROGRESSED_EVENT = "uploadProgressed";
+
+// @beta
+export type UploadCanceledListener = () => void;
+
+// @beta
+export type UploadCompleteListener = (metaData: FileMetaData) => void;
+
+// @beta
+export class UploadedFile implements UploadEvents {
+    constructor(file: File);
+    cancelUpload(): void;
+    completeUpload(metaData: FileMetaData): void;
+    extension(): string;
+    failUpload(message: string): void;
+    file: File;
+    // (undocumented)
+    isUploaded(): boolean;
```

Comment:
is seems slightly odd to have a public method `isUploaded()` when the user of this class has to do `completeUpload` for isUploaded to be true (i.e. class owner should have no need for this method as they had to mark the file as uploaded) -- perhaps make it a private method?

Reply:
This PR is stale now, let me update it with the latest changes and this will be resolved :)